.cargo/config.toml

```diff
@@ -1,15 +0,0 @@
-[build]
-#target = "wasm32-unknown-unknown"
-#target = "aarch64-apple-darwin"
-rustflags = ["--cfg", "web_sys_unstable_apis"]
```

Comment:
what about the cfg flag?

Reply:
Ah right.. maybe i'll add it to the examples directly